Settings.c

```diff
@@ -806,3 +806,25 @@ void Settings_setHeaderLayout(Settings* this, HeaderLayout hLayout) {
    this->hLayout = hLayout;
    this->changed = true;
 }
+
+static bool ProcessField_hasField(const ProcessField* fields, ProcessField field) {
+   if (fields) {
+      while (*fields) {
+         if (*fields++ == field) {
+            return true;
+         }
+      }
+   }
+   return false;
+}
```

Comment:
Use the `accumulated flags that each screen already has …`

Reply:
Done